commands/certmanager/post.go

```diff
@@ -0,0 +1,135 @@
+package certmanager
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/spf13/cobra"
+)
+
+var createProperties = sdkgo.CertificatePostPropertiesDto{}
+
+func CertPostCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace:  "certmanager",
+		Resource:   "certificates",
+		Verb:       "create",
+		Aliases:    []string{"c"},
+		ShortDesc:  "Add a new Certificate",
+		LongDesc:   "Use this command to add a Certificate.",
+		Example:    "ionosctl certificate-manager create",
+		PreCmdRun:  PreCmdPost,
+		CmdRun:     CmdPost,
+		InitClient: true,
+	})
+
+	cmd.AddStringFlag(CertName, "", "", "Specify name of the certificate (required either this or --certificate-name-path)")
```

Comment:
Could you please add some shorthand flags? They don't need really need to be defined as constants, but it would help to do so, since we usually want to keep track of which shorthand flag is bound to which flag